plugin/evm/vm.go

```diff
@@ -404,21 +420,6 @@ func (vm *VM) Initialize(
 		return err
 	}
 
-	var extDataHashes map[common.Hash]common.Hash
-	// Set the chain config for mainnet/fuji chain IDs
-	switch chainCtx.NetworkID {
-	case avalanchegoConstants.MainnetID:
-		extDataHashes = mainnetExtDataHashes
-	case avalanchegoConstants.FujiID:
-		extDataHashes = fujiExtDataHashes
-	}
-	vm.extensionConfig.BlockExtender = newBlockExtender(extDataHashes, vm)
```

Comment:
(note to self and other reviewers) These are moved to `plugin/evm/atomic/vm/vm.go`